### ISSUE TITLE
Public API?

### ISSUE THREAD

**Issue by tornadocontrib** [ethereum/node-crawler#56]
Hello,

Is there any public API available to fetch nodes rather than downloading the PG dump?

cc @MariusVanDerWijden 

Thanks.

**Comment by angaz** [comment]
Hi, currently, ethernets.io is running from github.com/angaz/node-crawler.

There is an API for [raw stats rows](https://hackmd.io/NltGg8_FT2iOXoN2P1hwKA).

For node details, what exactly do you want to know?

**Comment by tornadocontrib** [comment]
@angaz Ah that is great any plans to merge the feature to the upstream? And I think the documentation should be available on github as well

**Comment by angaz** [comment]
Yeah, I'm working on cleanup for upstreaming, and docs is on the list for that.

**Comment by MqllR** [comment]
Sorry to chime in here, but my question is a kind of similar ! Is there any plan or would you be willing to extend the API ? For example, I'd be interested to grab a list of enode addresses based on parameters: network, location, client, etc...

**Comment by angaz** [comment]
Hi @MqllR, is maybe this issue describing what you are looking for? https://github.com/ethereum/node-crawler/issues/57

**Comment by MqllR** [comment]
Oh damn, just one issue before! Moving my comment over there !
